### PR TITLE
Update docs for attaching webdriver to the launched UWP app with remote tool.

### PR DESCRIPTION
**Pull request by yanlingwang23**
AB#44758181

### CODE REVIEW

File: microsoft-edge/webview2/how-to/webdriver.md [hunk 1]
```diff
@@ -227,7 +227,28 @@ At `localhost:9222` above, the port number given on this line should match the p
 
 For more info about the `DebuggerAddress` property of the `EdgeOptions` object, see [EdgeOptions object](../../webdriver-chromium/capabilities-edge-options.md#edgeoptions-object).
 
-Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
+### For UWP apps, attach Microsoft Edge WebDriver to WebView2 with Remote Tools
+
+1. Once you have launched the UWP WebView2 app, navigate to `http://<Device Portal URL>/msedge` and identify the `browserProcessId` for the WebView2 process that you want to attach Microsoft Edge WebDriver to.  For example, in the following screenshot, navigating to `localhost:50080/msedge` displays the `browserProcessId` as `47860`.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:232]:
```suggestion
1. After you've launched the UWP WebView2 app, go to `http://<Device Portal URL>/msedge` and identify the `browserProcessId` for the WebView2 process that you want to attach Microsoft Edge WebDriver to.

   For example, in the following screenshot, navigating to `localhost:50080/msedge` displays the `browserProcessId` as `47860`:
```

File: microsoft-edge/webview2/how-to/webdriver.md [hunk 2]
```diff
@@ -227,7 +227,28 @@ At `localhost:9222` above, the port number given on this line should match the p
...
+
+![ProcessId_of running UWP_app](../media/webdriver/sample-processId.png)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:234]:
```suggestion
   ![ProcessId of a running UWP app](../media/webdriver/sample-processid.png)
```
note: lowercased filename per Docs Contributor Guide > filenames; naming files

File: microsoft-edge/webview2/media/webdriver/sample-processId.png
Comment (mikehoffms) [inline microsoft-edge/webview2/media/webdriver/sample-processId.png:1]: rename to 
sample-processid.png
lowercase per Docs Contributor Guide

File: microsoft-edge/webview2/how-to/webdriver.md [hunk 3]
```diff
@@ -227,7 +227,28 @@ At `localhost:9222` above, the port number given on this line should match the p
...
+
+1. Now, use the `wdpAddress` and `wdpProcessId` properties to tell Microsoft Edge WebDriver to connect to the the [Remote Tools for Microsoft Edge](https://apps.microsoft.com/store/detail/9P6CMFV44ZLT) and the specific WebView2 process. `wdpAddress` is defined as the Device Portal URL and `wdpProcessId` is the `browserProcessId` you identified in the previous step.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:236]:
```suggestion
1. Use the `wdpAddress` and `wdpProcessId` properties to tell Microsoft Edge WebDriver to connect to the the [Remote Tools for Microsoft Edge](https://apps.microsoft.com/store/detail/9P6CMFV44ZLT) and the specific WebView2 process.  `wdpAddress` is defined as the Device Portal URL.  `wdpProcessId` is the `browserProcessId` that you identified in the previous step:
```

File: microsoft-edge/webview2/how-to/webdriver.md [hunk 4]
```diff
@@ -227,7 +227,28 @@ At `localhost:9222` above, the port number given on this line should match the p
...
+```csharp
+EdgeOptions eo = new EdgeOptions();
+eo.AddAdditionalEdgeOption("wdpAddress", "localhost:50080");
+eo.AddAdditionalEdgeOption("wdpProcessId", 47860);
+// Optional user name and password to use when connecting to a Windows Device Portal server.
+// Required if the server has authentication enabled.
+// eo.AddAdditionalEdgeOption("wdpUsername", "username");
+// eo.AddAdditionalEdgeOption("wdpPassword", "password");
+EdgeDriver e = new EdgeDriver(eo);
+```
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:247]:
```suggestion
   ```csharp
   EdgeOptions eo = new EdgeOptions();
   eo.AddAdditionalEdgeOption("wdpAddress", "localhost:50080");
   eo.AddAdditionalEdgeOption("wdpProcessId", 47860);
   // Optional user name and password to use when connecting to a Windows Device Portal
   // server.  Required if the server has authentication enabled.
   // eo.AddAdditionalEdgeOption("wdpUsername", "username");
   // eo.AddAdditionalEdgeOption("wdpPassword", "password");
   EdgeDriver e = new EdgeDriver(eo);
   ```
```

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/webdriver.md:247]: * shifted to col4 not col1, to match "Use" in above sentence
* wrapped (comment line) at ~col 85 per Docs Contributor Guide, confirmed previously on 15" laptop @ 100% as a good width.

File: microsoft-edge/webview2/how-to/webdriver.md [hunk 5]
```diff
@@ -227,7 +227,28 @@ At `localhost:9222` above, the port number given on this line should match the p
...
+
+For more info about launching WV2 UWP app with Remote Tools, see [Remote debugging UWP apps with Remote Tools for Microsoft Edge](./remote-debugging.md).
```
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:249]:
```suggestion
For more information about launching a WebView2 UWP app with Remote Tools, see [Remote debugging UWP apps with Remote Tools for Microsoft Edge](./remote-debugging.md).
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor pass

File: microsoft-edge/webview2/how-to/webdriver.md (hunk 2 above)
Comment (mikehoffms) [inline microsoft-edge/webview2/how-to/webdriver.md:234]:
```suggestion
1. After you've launched the UWP WebView2 app, go to `http://<Device Portal URL>/msedge`.  For example, the following screenshot shows `localhost:50080/msedge`:

   ![ProcessId of a running UWP app](../media/webdriver/sample-processid.png)

1. Identify the `browserProcessId` for the WebView2 process that you want to attach Microsoft Edge WebDriver to.  For example, the above screenshot shows the `browserProcessId` as `47860`.
```

File: microsoft-edge/webview2/media/webdriver/sample-processId.png
Comment (mikehoffms) [inline microsoft-edge/webview2/media/webdriver/sample-processId.png:1]: delete this unused png

**Review by mikehoffms — APPROVED** [review]

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/webview2/media/webdriver/sample-processId.png
Comment (mikehoffms) [reply microsoft-edge/webview2/media/webdriver/sample-processId.png:1]: this thread can be resolved

Comment (mikehoffms) [reply microsoft-edge/webview2/media/webdriver/sample-processId.png:1]: this thread can be resolved

**Pull request merged**